fix linking order of libraries to fix:
  ld: /software/LibTIFF/4.1.0-GCCcore-10.2.0/lib/libtiff.so:(*IND*+0x0): multiple definition of `_end'
  ld: /software/LibTIFF/4.1.0-GCCcore-10.2.0/lib/libtiff.so:(*IND*+0x0): multiple definition of `__bss_start'
see also https://github.com/easybuilders/easybuild-easyconfigs/issues/14219
--- imread-0.7.1/setup.py.orig  2022-02-25 17:30:06.726515267 +0100
+++ imread-0.7.1/setup.py       2022-02-25 17:30:21.451056892 +0100
@@ -77,7 +77,7 @@
 }


-libraries = ['png', 'jpeg', 'tiff']
+libraries = ['png', 'tiff', 'jpeg']
 if sys.platform.startswith('win'):
     libraries.append('zlib')

